File: php/memory.php
<?php
    include "product.php";

    class memory extends product{

        private $freq = "";
        private $memSize = "";
        private $spCuda = "";

        public function __construct($freq="", $memSize="", $spCuda=""){
            $this->freq = $freq;
            $this->memSize = $memSize;
            $this->spCuda = $spCuda;
        }

        public function setFreq($freq){
            $this->freq = $freq;
        }

        public function getFreq(){
            return $this->freq;
        }

        public function setMemSize($memSize){
            $this->memSize = $memSize;
        }

        public function getMemSize(){
            return $this->memSize;
        }

        public function setSpCuda($spCuda){
            $this->spCuda = $spCuda;
        }

        public function getSpCuda(){
            return $this->spCuda;
        }
    }

    function __destruct(){
    }
?>

File: python/Memory.py
#mengimport Product sebagai ortunya (inheritance)
from Product import Product

class Memory(Product):
    __freq = ''
    __memSize = ''
    __spCuda = ''

    def __init__(self, freq='', memSize='', spCuda=''):
        self.__freq = freq
        self.__memSize = memSize
        self.__spCuda = spCuda

    def setFreq(self, freq):
        self.__freq = freq
    
    def getFreq(self):
        return self.__freq

    def setMemSize(self, memSize):
        self.__memSize = memSize
    
    def getMemSize(self):
        return self.__memSize

    def setSpCuda(self, spCuda):
        self.__spCuda = spCuda
    
    def getSpCuda(self):
        return self.__spCuda
